### ISSUE TITLE
ADDON_ACTION_BLOCKED: CompactRaidFrame

### ISSUE THREAD

**Issue by Slackluster** [Caerdon/CaerdonWardrobe#22]
Alright, so this time I've tested it with no other addons enabled. Just Caerdon, BugGrabber, and BugSack. The following error pops up, or similar ones all related to the CompactRaidFrame:

```
1x [ADDON_ACTION_BLOCKED] AddOn 'CaerdonWardrobe' tried to call the protected function 'CompactRaidFrame2:ClearAllPoints()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `ClearAllPoints'
[string "@FrameXML\FlowContainer.lua"]:157: in function `FlowContainer_DoLayout'
[string "@FrameXML\FlowContainer.lua"]:17: in function `FlowContainer_ResumeUpdates'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:185: in function `CompactRaidFrameContainer_LayoutFrames'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:130: in function `CompactRaidFrameContainer_TryUpdate'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:57: in function `CompactRaidFrameContainer_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

Locals:
Skipped (In Encounter)
```

It seems to appear both grouped and ungrouped, when doing instances, but I've not been able to properly recreate it. It just... sometimes shows up. I could be pulling trash, or just killing something, or nothing in particular.

Apparently this is a "taint" and can be ignored, from what I can find on forums and such, but because I'm an idiot that doesn't know what that means I just thought I should report it here anyway. I know it does refer to code in BugGrabber itself, which I do not understand, but yeah.

**Comment by Caerdon** [comment]
Thanks for the follow up!  While I haven't been able to reproduce the error in game, I spent some time digging into the taint logs to see what I could discover, and I think this may have been introduced when I made improvements to the addon settings screen.  I'm looking at options to mitigate and will hopefully have some time soon to resolve!

**Comment by Caerdon** [comment]
I landed some changes in v2.22.0 that got rid of the taint I was seeing.  Give it a shot and see if it helps and feel free to reopen if not!
